chapters/solutions.Rmd

```diff
@@ -62,6 +62,18 @@ The files/directories in each directory are sorted by time of last change.
     efficiently generate a blank text file to be used by such
     programs.
 
+### Exercise \@ref(bash-basics-ex-safe-rm) {-}
+
+```shell
+$ rm: remove regular file 'thesis_backup/quotations.txt'? y
+```
+
+The `-i` option will prompt before (every) removal
```

Comment:
I'm pretty sure this is the first time readers will have seen the command line prompt for an answer this way; I'd add a sentence that explicitly states to hit enter to select yes.

Reply:
j/k, I see it in the line below